features/steps/given.php

```diff
@@ -127,12 +127,18 @@ function ( $world, $stream, $output_filter, $key ) {
 	}
 );
 
-$steps->Given( '/^a new Phar(?: with version "([^"]+)")$/',
-	function ( $world, $version ) {
+$steps->Given( '/^a new Phar(?: with version "([^"]+)")?$/',
```

Comment:
I think this would be better as `a new Phar with (the same version|version "([^"]+)")` so it's more literal in the tests